billy_metadata/de.py

```diff
@@ -0,0 +1,128 @@
+            'sessions': ['141'],
+        },
+        {
+            'name': '2003-2004',
+            'start_year': 2003,
+            'end_year': 2004,
+            'sessions': ['142'],
+        },
+        {
+            'name': '2005-2006',
+            'start_year': 2005,
+            'end_year': 2006,
+            'sessions': ['143'],
+        },
+        {
+            'name': '2007-2008',
+            'start_year': 2007,
+            'end_year': 2008,
+            'sessions': ['144'],
+        },
+        {
+            'name': '2009-2010',
+            'start_year': 2009,
+            'end_year': 2010,
+            'sessions': ['145'],
+        },
+        {
+            'name': '2011-2012',
+            'start_year': 2011,
+            'end_year': 2012,
+            'sessions': ['146'],
+        },
+        {
+            'name': '2013-2014',
+            'start_year': 2013,
+            'end_year': 2014,
+            'sessions': ['147'],
+        },
+        {
+            'name': '2015-2016',
+            'start_year': 2015,
+            'end_year': 2016,
+            'sessions': ['148'],
+            },
+        {
+            'name': '2017-2018',
+            'start_year': 2017,
+            'end_year': 2018,
+            'sessions': ['149'],
+            },
+    ],
+    'session_details': {
+         '140': {
+            'display_name': '140th General Assembly (1999-2000)',
+            '_scraped_name': '1998 - 2000 (GA 140)',
+        },
+         '141': {
+            'display_name': '141st General Assembly (2001-2002)',
+            '_scraped_name': '2000 - 2002 (GA 141)',
+        },
+         '142': {
+            'display_name': '142nd General Assembly (2003-2004)',
+            '_scraped_name': '2002 - 2004 (GA 142)',
+        },
+         '143': {
+            'display_name': '143rd General Assembly (2005-2006)',
+            '_scraped_name': '2004 - 2006 (GA 143)',
+        },
+        '144': {
+            'display_name': '144th General Assembly (2007-2008)',
+            '_scraped_name': '2006 - 2008 (GA 144)',
+        },
+        '145': {
+            'display_name': '145th General Assembly (2009-2010)',
+            '_scraped_name': '2008 - 2010 (GA 145)',
+        },
+        '146': {
+            'display_name': '146th General Assembly (2011-2012)',
+            '_scraped_name': '2010 - 2012 (GA 146)',
+        },
+        '147': {
+            'display_name': '147th General Assembly (2013-2014)',
+            '_scraped_name': '2012 - 2014 (GA 147)',
+        },
+        '148': {
+            'display_name': '148th General Assembly (2015-2016)',
+            '_scraped_name': '2014 - 2016 (GA 148)',
+        },
+        '149': {
+            'display_name': '149th General Assembly (2017-2018)',
+            '_scraped_name': '2016 - 2018 (GA 149)',
+        },
+    },
+    'feature_flags': ['events', 'influenceexplorer'],
+    '_ignored_scraped_sessions': [
+
+    ],
+}
+
+
+def session_list():
```

Comment:
Should this be moved to `Delaware.get_session_list` in `openstates/de/__init__.py`?